src/pencilkit.cs

```diff
@@ -246,9 +300,171 @@ interface PKToolPicker {
 		[Export ("colorUserInterfaceStyle", ArgumentSemantic.Assign)]
 		UIUserInterfaceStyle ColorUserInterfaceStyle { get; set; }
 
+		[Introduced (PlatformName.iOS, 13, 0, message: "Create individual instances instead.")]
+		[Deprecated (PlatformName.iOS, 14, 0, message: "Create individual instances instead.")]
 		[Static]
 		[return: NullAllowed]
 		[Export ("sharedToolPickerForWindow:")]
 		PKToolPicker GetSharedToolPicker (UIWindow window);
+
+		[iOS (14, 0)]
+		[Introduced (PlatformName.MacCatalyst, 14, 0)]
+		[Export ("showsDrawingPolicyControls")]
+		bool ShowsDrawingPolicyControls { get; set; }
+
+		// !missing-null-allowed! 'System.Void PencilKit.PKToolPicker::set_AutosaveName(System.String)' is missing an [NullAllowed] on parameter #0
+		[iOS (14, 0)]
+		[Introduced (PlatformName.MacCatalyst, 14, 0)]
+		[Export ("stateAutosaveName")]
+		string stateAutosaveName { get; set; }
+	}
+
+	[Mac (10, 16), iOS (14, 0)]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
+	[BaseType (typeof(NSObject))]
+	interface PKInk : NSCopying
+	{
+		[Export ("initWithInkType:color:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string type, UIColor color);
+
+		[Export ("inkType")]
+		string InkType { get; }
+
+		[Export ("color")]
+		UIColor Color { get; }
+	}
+
+	[Mac (10, 16), iOS (14, 0)]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
+	[BaseType (typeof(NSObject))]
+	interface PKFloatRange : NSCopying
+	{
+		[Export ("initWithLowerBound:upperBound:")]
+		IntPtr Constructor (nfloat lowerBound, nfloat upperBound);
+
+		[Export ("lowerBound")]
+		nfloat LowerBound { get; }
+
+		[Export ("upperBound")]
+		nfloat UpperBound { get; }
+	}
+
+	[Mac (10, 16), iOS (14, 0)]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
+	[BaseType (typeof(NSObject))]
+	interface PKStroke : NSCopying
+	{
+		[NoMac]
+		[Export ("initWithInk:strokePath:transform:mask:")]
+		IntPtr Constructor (PKInk ink, PKStrokePath path, CGAffineTransform transform, [NullAllowed] UIBezierPath mask);
+
+		[NoiOS]
+		[Export ("initWithInk:strokePath:transform:mask:")]
+		IntPtr Constructor (PKInk ink, PKStrokePath path, CGAffineTransform transform, [NullAllowed] NSBezierPath mask);
+
+		[Export ("ink")]
+		PKInk Ink { get; }
+
+		[Export ("transform")]
+		CGAffineTransform Transform { get; }
+
+		[Export ("path")]
+		PKStrokePath Path { get; }
+
+#if MONOMAC
```

Comment:
I thought that the #if/#else wouldn't be needed with the `using` statements at the top of this file, but it looks like c# is unable to distinguish between these two signatures for `Mask` w/o the `using` :(

```
GEN      [ios] generated_sources
error BI0002: bgen: Could not compile the API bindings.
		iosurface.cs(202,4): warning CS0657: 'return' is not a valid attribute location for this declaration. Valid attribute locations for this declaration are 'property'. All attributes in this block will be ignored.
	pencilkit.cs(382,16): error CS0102: The type 'PKStroke' already contains a definition for 'Mask'
```

Is there a nicer way to do this?

Reply:
I think you can do better than doing the #if here. On monomac we could simply point to the correct type via a using, similar to what we do here: https://github.com/xamarin/xamarin-macios/blob/main/src/UIKit/NSLayoutManager.cs#L24

Reply:
Yes, you can do this at the top:

```csharp
#if MONOMAC
using BezierPath = AppKit.NSBezierPath;
#else
using BezierPath = UIKit.UIBezierPath;
#endif
```

and then here you can just do:

```csharp
[NullAllowed, Export ("mask")]
BezierPath Mask { get; }
```

and the same for the constructor above as well.